### PR TITLE
feat(shared-data): add 1-well troughs and 96-well deep well plate

### LINKED ISSUE #3555: Labware Definition: 1 Well Trough
PLACEHOLDER

As an Opentrons user, I would like to be able to use a 1-well trough in my protocols.

## Acceptance Criteria
- TBD, need to identify appropriate 1-well trough and add to spreadsheet.

### PR DESCRIPTION
**Pull request by ahiuchingau**
## overview
Closes #3555 by adding definition of Agilent 1-well trough, Axygen 1-well trough and USA Scientific 96-well 2.4 mL deep well plate based on spreadsheet

## review requests
Please let me know if anything's missing

### CODE REVIEW

File: shared-data/labware/definitions/2/agilent_1_reservoir_290ml/1.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "ordering": [["A1"]],
+  "brand": {
+    "brand": "Agilent",
+    "brandId": ["201252-100"]
```
Comment (mcous) [inline shared-data/labware/definitions/2/agilent_1_reservoir_290ml/1.json]: Can we add a "links" array with the purchase link from the spreadsheet?

File: shared-data/labware/definitions/2/axygen_1_reservoir_90ml/1.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+  "ordering": [["A1"]],
+  "brand": {
+    "brand": "Axygen",
+    "brandId": ["RES-SW1-LP"]
```
Comment (mcous) [inline shared-data/labware/definitions/2/axygen_1_reservoir_90ml/1.json]: Same comment about `"links"`

File: shared-data/labware/definitions/2/usascientific_96_wellplate_2.4ml_deep/1.json [hunk 3]
```diff
@@ -0,0 +1,1113 @@
...
+    ["A7", "B7", "C7", "D7", "E7", "F7", "G7", "H7"],
+    ["A8", "B8", "C8", "D8", "E8", "F8", "G8", "H8"],
+    ["A9", "B9", "C9", "D9", "E9", "F9", "G9", "H9"],
+    ["A10", "B10", "C10", "D10", "E10", "F10", "G10", "H10"],
+    ["A11", "B11", "C11", "D11", "E11", "F11", "G11", "H11"],
+    ["A12", "B12", "C12", "D12", "E12", "F12", "G12", "H12"]
+  ],
+  "brand": {
+    "brand": "USA Scientific",
+    "brandId": ["1896-2000"]
```
Comment (mcous) [inline shared-data/labware/definitions/2/usascientific_96_wellplate_2.4ml_deep/1.json]: links

File: shared-data/labware/definitions/2/usascientific_96_wellplate_2.4ml_deep/1.json [hunk 4]
```diff
@@ -0,0 +1,1114 @@
...
+  },
+  "metadata": {
+    "displayName": "USA Scientific 96 Well Plate 2.4 mL",
```
[the remark this answers is not in the document]
Comment (ahiuchingau) [reply shared-data/labware/definitions/2/usascientific_96_wellplate_2.4ml_deep/1.json]: It should, so I'm also adding that to display name of `generic_96_wellplate_1.5ml_deep`

File: shared-data/labware/definitions/2/agilent_1_reservoir_290ml/1.json [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  },
+  "wells": {
+    "A1": {
+      "depth": 39.22,
+      "shape": "rectangular",
+      "xDimension": 108,
+      "yDimension": 72,
+      "totalLiquidVolume": 290000,
+      "x": 63.88,
+      "y": 11.24,
```
Comment (mcous) [inline shared-data/labware/definitions/2/agilent_1_reservoir_290ml/1.json]: Should this be `42.785`?

**Pull request merged**